Applications/Games/Space Colony/Local/script.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Local",
+    "id"                         : "space_colony_local",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
+    "free"                    : false,
+    "requiresPatch"           : false
```

Comment:
"requiresPatch" should probably be `true`.

Reply:
Not true. According to this:
https://phoenicisorg.github.io/scripts/Develop/script-json/
this vaule should be set to "true" only if the game will not work due to protection algorythms.

Reply:
You're right.